modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/buttons/RowButton.kt

```diff
@@ -37,6 +39,79 @@ fun RowButton(
     @DrawableRes leadingIcon: Int? = null,
     onClick: () -> Unit
 ) {
+
+    BaseRowButton(
+        text = text,
+        modifier = modifier,
+        includePadding = includePadding,
+        enabled = enabled,
+        border = border,
+        colors = colors,
+        textColor = textColor,
+        leadingIcon = if (leadingIcon != null) {
+            {
+                Icon(
+                    painter = painterResource(leadingIcon),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .padding(4.dp),
+                    tint = textColor,
+                )
+            }
+        } else null,
+        onClick = onClick,
+    )
+}
+@Composable
+fun RowLoadingButton(
```

Comment:
I noticed previews for `RowButton` were replaced with `RowLoadingButton` in the previews here:

https://github.com/Automattic/pocket-casts-android/pull/1474/commits/a0ee61ff61869191e44066e2b153a7b818a7c7a7#diff-37cb97d49ec540989482b482c81a88357f6a92c95bb309eac181a6a5a7222fd4L79-R185

I reverted them while resolving the conflict as we would like to keep the `RowButton` previews. Can we add a preview for just the `RowLoadingButton`?

Reply:
I'll add that too!

Reply:
Done

Reply:
Thanks! I just enabled isLoading in the previews so that loading icon can be seen in Profile -> Settings -> Developer -> Showkase -> Button. It looks great!

<img width=200 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/98675a11-c2ca-428d-96ff-d0bf753735b2"/>

Reply:
Awesome. Thanks!